index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+       
+  <title>ILC 5</title>
+  <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css">
+  <link rel="stylesheet" href="css/style.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
+  <script src="js/accounting.min.js"></script>
+  <script src="js/script.js"></script>
+  <meta charset="utf-8"> 
+</head>
+<button id="get-employees" class="centered">Click me to get employees</button>
+<br>
+<div id="loader" style="display:nome;" class="centered"><image src="img/image.gif"/></div>
```

Comment:
style should be display: none; - you have a typo which is why your loader shows all the time.